### ISSUE TITLE
The summary provides information per CVE number, however it would be good for WSUS users to have this information per KB

### ISSUE THREAD

**Issue by chilleregeravi** [microsoft/MSRC-Microsoft-Security-Updates-API#31]
The output of Get-MsrcCvrfDocument cmdlet is with key as CVE number. This is good from a security point of view that all CVE's are covered. However, for a WSUS Admin point of view, it would KBs that are more important. 

A particular KB has more than one CVE linked to it. Hence, we should have a cmdlet which generates report per KBnumber with information on the CVEs covered by it.

**Comment by jm0477** [comment]
Agreed. actually the KBs are not unique. the URL given from the CVRF documents, it's only the catalog+queryKB.  when accessing the catalog, the results will be multiple. we cannot get the exact one we want.

**Comment by chilleregeravi** [comment]
Any update on this enhancement?

**Comment by karanatpanaseer** [comment]
Hello, is there any update on this? What's the best way to get data indexed by KB rather than by CVE ?

Thank you in advance
